emperor/support_files/js/emperor.js

```diff
@@ -739,6 +738,14 @@ function colorChangedForTaxaSpheres(color){
 	}
 }
 
+function colorChangedForTaxaLabels(color){
+        // get the taxonomic assignments and append '_taxalabel' to
+        // retrieve all the labels belonging to a sphere in the plot
+        for(var key in g_taxaPositions) {
```

Comment:
I'm reworking this file so it uses better ways to loop by using _ (underscorejs), could you change this to:
```
_.each(g_taxaPositions, function(taxaPositions, key){
  $('#'+key+"_taxalabel").css('color', color);
});
```